<launch>
	<param name="dataFolder" value="/tmp/ssl_robot_data" />
	<node name="affw_export" pkg="affw_ctrl" type="affw_export" output="screen" />
	<node name="ssl_robot_export" pkg="ssl_robot_affw" type="ssl_robot_export" output="screen" />
</launch>